pywbem_mock/_mainprovider.py

```diff
@@ -2506,7 +2508,19 @@ def _open_response(self, namespace, objects, pull_type, OperationTimeout,
             # remove objects from list that are being sent to client
             del objects[0: max_obj_cnt]
 
-        return(rtn_objects, eos, context_id)
+        return (rtn_objects, eos, context_id)
+
+    def _openquery_response(self, namespace, objects, pull_type,
+                            OperationTimeout, MaxObjectCount, ContinueOnError,
+                            QueryResultClass):
+        """
+        Like _openquery_response(), just adds QueryResultClass to the returned
```

Comment:
I assume you meant "Like _open_response()"

Reply:
DONE.